ifopt_core/CMakeLists.txt

```diff
@@ -1,20 +1,21 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(ifopt_core)
+option(CATKIN_BUILD "CATKIN_BUILD" ON)
```

Comment:
This could be set from the return value of 
`find_package(catkin` QUIET)`. That way the user doesn't have to worry about it